src/data/quality-assurance-quiz.ts

```diff
@@ -182,6 +182,58 @@ const qualityAssuranceQuiz = [
     Explanation:
       "A good QA engineer should understand the full life cycle of software products in order to effectively plan and test them. There are 7 phases of the Software Development Life Cycle",
     Link: "https://www.freecodecamp.org/news/how-to-become-a-quality-assurance-engineer-qa-engineer-career-guide/"
+  },
+
+  {
+    Question:
+      "What is the difference between 'functional testing' and 'non-functional testing' in software testing? Provide examples of each.",
+    Answer:
+      "'Functional testing' assesses the functionality of a software application by testing individual functions or features. Examples include unit testing and integration testing. 'Non-functional testing' evaluates non-functional aspects like performance, security, and usability. Examples include load testing, security testing, and usability testing.",
+    Distractor1:
+      "Functional testing and non-functional testing are the same thing.",
+    Distractor2:
+      "Functional testing only involves manual testing, while non-functional testing uses automation.",
+    Distractor3: "Non-functional testing is limited to web applications.",
+    Explanation:
+      "Functional testing ensures that the software meets specified requirements, while non-functional testing assesses how well the software performs under various conditions.",
+    Link: ""
```

Comment:
the link field should always be filled out

```suggestion
    Link: "https://www.freecodecamp.org/news/software-testing-beginners-guide/"
```